radar/data_management/kafka.yaml

```diff
@@ -7,15 +7,28 @@ description: |
   developed by LinkedIn and donated to the Apache Software Foundation, written in Scala and Java.
   The project aims to provide a unified, high-throughput,
   low-latency platform for handling real-time data feeds.
-rationale:
-  Kafka will cover several improvement needs in Retail Suite,
-  and external systems communicating with Retail Suite.
-  Its inherient scalability will allow us to target the Tier-1 and Tier-2 customers
-  and support their traffic and computational needs in the daily operation of Retail Suite.
-  The built in commit log can be regarded as the source-of-truth for several entities.
-  The power of real-time streaming can enable Retail Suite to process vast amounts of data faster
-  and more precise than today.
+rationale: |
+  We consider Kafka a streaming platform to be used internally within a microservice domain
+  when stream processing capabilities not supported by the general purpose messaging platform.
+  To use Kafka in a service, the following conditions must be met:
+
+    * Kafka is not our messaging platform, if no stream processors are needed,
+      use the general purpose message platform
+    * Use of Kafka is private to the clan's services. For all intents, consider
+      Kafka the equal of a database
+    * A business case can be built around the cost driven by using Kafka
+    * Kafka topics are internal API and cannot be used to share messages with
+      services with services maintained by other clans
```

Comment:
This bulletpoint can be removed as the messaging part has already been stated in the first bulletpoint.

Reply:
I'll rewrite this a bit and incorporate the info in the above bullet. I want to emphasize that topics are private API to avoid potential confusion.

Reply:
Well, I am not sure i agree with that. Are Topics in PubSub private to the clan? Does that mean that if POS teams post ARTS sales to a topic, CIM will not be allowed to read it?

Reply:
We use the messaging platform for that data exchange, not Kafka topics.